src/rowtransforms.jl

```diff
@@ -0,0 +1,60 @@
+struct NormalizeRow <: DataAugmentation.Transform
+	normstats
+	normcols
+end
+
+struct Categorify <: DataAugmentation.Transform
+	pooldict
+	categorycols
+end
+
+struct FillMissing <: DataAugmentation.Transform
+	fmvals
+	contcols
+	catcols
+end
+
+function DataAugmentation.apply(tfm::FillMissing, item::TabularItem; randstate=nothing)
+	x = (; zip(item.columns, [data for data in item.data])...)
+	for col in tfm.contcols
+		if ismissing(x[col])
+			Setfield.@set! x[col] = tfm.fmvals[col]
+		end
+	end
+	for col in tfm.catcols
+		if ismissing(x[col])
+			Setfield.@set! x[col] = "missing"
+		end
+	end
+	TabularItem(x, item.columns)
+end
+
+function DataAugmentation.apply(tfm::NormalizeRow, item::TabularItem; randstate=nothing)
+	x = (; zip(item.columns, [data for data in item.data])...)
+	for col in tfm.normcols
```

Comment:
Instead of iterating the columns twice and having setfield repeatedly construct a namedtuple, perhaps look into a helper function that does the normalization given the `tfm`, column name and value? The function could check if the column is in `normcols` and transform it with `normstats` if it is.

Reply:
Yeah do we even need Setfield anymore if we're standardizing on a `NamedTuple`? We could just build the transformed data as a vector or something then construct a `NamedTuple` at the end.

Reply:
Yeah we should be able to build it at the end.